spacepy/coordinates.py

```diff
@@ -391,3 +397,81 @@ def append(self, other):
         data.extend(list(otherdata.data))
         newobj = Coords(data, dtype=self.dtype, carsph=self.carsph)
         return newobj
+
+    # -----------------------------------------------
+    def to_skycoord(self):
+        '''
+        Create an Astropy SkyCoord instance based on this instance
+
+        Returns
+        =======
+        out : Astropy SkyCoord instance
+            This coordinate as an Astropy SkyCoord
+
+        Notes
+        =====
+        This method requires Astropy to be installed.
+
+        This method uses the GEO coordinate frame as the common frame between the two libraries.
+        '''
+        try:
+            from astropy.coordinates import SkyCoord
+            from astropy.time import Time
+        except ImportError as e:
+            raise e.__class__("This method requires Astropy to be installed.")
```

Comment:
It's good to propagate the error forward:
```suggestion
            raise e.__class__("This method requires Astropy to be installed.") from e
```

Reply:
Done

Reply:
Reverted.  SpacePy still tests against Python 2.7, where this is not valid syntax.

Reply:
Does it still test against 2.7?

Reply:
As far as I can tell, SpacePy still tries to support 2.7